minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java

```diff
@@ -238,14 +239,10 @@ protected void setupTest() throws IOException {
     UnderFileSystemUtils
         .touch(PathUtils.concatPath(journalFolder, "_format_" + System.currentTimeMillis()));
 
-    // If we are using the LocalMiniDFSCluster or S3UnderStorageCluster or OSSUnderStorageCluster,
+    // If we are using anything except LocalFileSystemCluster as UnderFS,
     // we need to update the UNDERFS_ADDRESS to point to the cluster's current address.
     // This must happen after UFS is started with UnderFileSystemCluster.get().
-    // TODO(andrew): Move logic to the alluxio-tests module so that we can use instanceof here
-    // instead of comparing classnames.
-    if (mUfsCluster.getClass().getSimpleName().equals("LocalMiniDFSCluster")
-        || mUfsCluster.getClass().getSimpleName().equals("S3UnderStorageCluster")
-        || mUfsCluster.getClass().getSimpleName().equals("OSSUnderStorageCluster")) {
+    if (!UnderFileSystemCluster.getUnderFSClass().equals(LocalFileSystemCluster.class.getName())) {
```

Comment:
could this just be 

```
if (!mUfsCluster.getClass().getSimpleName().equals(LocalFileSystemCluster.class.getSimpleName()) {
  ...
}
```

which would not require changed to `UnderFileSystemCluster` ... your change of setting `sUnderFsClass` to `LocalFileSystemCluster` when it is null is subtle and I would prefer to avoid it